WooCommerce/Classes/ViewRelated/Search/SearchCommand.swift

```diff
@@ -0,0 +1,33 @@
+import UIKit
+
+/// An interface for search UI associated with a generic model and cell view model.
+protocol SearchCommand {
```

Comment:
It seems like this does more than just being one command, and it's more like a ViewModel. Would it make sense to rename it to reflect that?

Reply:
I renamed it to `SearchUICommand` in https://github.com/woocommerce/woocommerce-ios/pull/1306/commits/0a260bc7bd0f2dd0931cb83473583dce77004b6a - lemme know if there's a better name for it!